### DISCUSSION TITLE
Stop F1 Help on Windows

### DISCUSSION THREAD

**Discussion by theoreaves** [alpinejs/alpine#2549] [1. Help] [answered]
I have a need to capture the F1 key and perform an event.  I added this code to my x-data element:

```
@keyup.f1.window.prevent.stop="$dispatch('new', {siID: '0' })"
```

This works as expected on a Mac.  However, on a windows machine, I keep getting the help window.
I looked at some raw javascript where I've done this in the past, which works and it looks like this:

```
if (e.keyCode==112){
  if (browser=="Netscape"){
   e.stopPropagation()
   e.preventDefault()
   } else {
   e.returnValue = false;
   e.keyCode = 0;
  }
   send_colon(':F1');
}
```

I've tried putting both stop and prevent on my alpine line (together and separate) but no luck.

So... Any suggestions would be great.

Thanks!

**Comment by SimoTod** [comment, marked as answer]
I think Windows listens for the keydown event. See https://codepen.io/SimoTod/pen/KKXodow

**Reply by theoreaves** [reply]
Indeed, that was it.  So simple, why didn't I try that?  LOL!

Thank you!
